Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -105,3 +142,47 @@ Cilium including an etcd cluster managed by the cilium-etcd-operator.
     .. parsed-literal::
 
       kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/1.8/cilium-with-node-init.yaml
+
+
+Restart remaining pods
+======================
+
+Once Cilium is up and running, restart all pods in ``kube-system`` so they can
+be managed by Cilium, similar steps that we have previously performed for ``kube-dns``
+
+::
+
+    $ kubectl get pods --all-namespaces -o wide
+    kube-system   event-exporter-v0.2.3-85644fcdf-9x77g                2/2     Running            0          30m   10.56.0.7     gke-cluster1-default-pool-a63a765c-flr2   <none>
+    kube-system   fluentd-gcp-scaler-8b674f786-6vwfc                   1/1     Running            0          30m   10.56.0.2     gke-cluster1-default-pool-a63a765c-flr2   <none>
+    kube-system   fluentd-gcp-v3.2.0-9ck4p                             2/2     Running            0          29m   10.56.1.6     gke-cluster1-default-pool-a63a765c-z73c   <none>
+    kube-system   fluentd-gcp-v3.2.0-xhjwq                             2/2     Running            0          29m   10.56.0.10    gke-cluster1-default-pool-a63a765c-flr2   <none>
+    kube-system   heapster-v1.6.0-beta.1-8f4db6558-vdqgg               2/3     CrashLoopBackOff   6          29m   10.56.1.4     gke-cluster1-default-pool-a63a765c-z73c   <none>
+    kube-system   kube-dns-548976df6c-ckm2l                            4/4     Running            0          22m   10.56.1.245   gke-cluster1-default-pool-a63a765c-z73c   <none>
+    kube-system   kube-dns-548976df6c-fz6gz                            4/4     Running            0          22m   10.56.0.151   gke-cluster1-default-pool-a63a765c-flr2   <none>
+    kube-system   kube-dns-autoscaler-67c97c87fb-frvqj                 1/1     Running            0          30m   10.56.0.4     gke-cluster1-default-pool-a63a765c-flr2   <none>
+    kube-system   kube-proxy-gke-cluster1-default-pool-a63a765c-flr2   1/1     Running            0          30m   10.164.0.2    gke-cluster1-default-pool-a63a765c-flr2   <none>
+    kube-system   kube-proxy-gke-cluster1-default-pool-a63a765c-z73c   1/1     Running            0          30m   10.164.0.3    gke-cluster1-default-pool-a63a765c-z73c   <none>
+    kube-system   l7-default-backend-7ff48cffd7-7qmv9                  1/1     Running            0          22s   10.56.0.69    gke-cluster1-default-pool-a63a765c-flr2   <none>
+    kube-system   metrics-server-v0.2.1-fd596d746-x98pr                2/2     Running            0          29m   10.56.1.5     gke-cluster1-default-pool-a63a765c-z73c   <none>
+
+You can choose to specify all pod names manually with ``kubectl delete pod -n kube-system <pod1> <pod2> ...``
```

Comment:
This description makes no sense. You instruct to restart all pods in kube-system, you then suggest to list all pods but mentioned that if `--all` is used, kube-proxy is restarted unnecessarily. The user would also list kube-proxy based on your instructions though. What pods do we have to restart for real? Maybe we should provide a command that filters non-host-networking pods and restarts them. This is making the entire instructions look very complicated.